### PR TITLE
added some agile quiz questions

### PR DESCRIPTION
**Pull request by SWASTI-SUNDAR**
## Summary of changes
added some agile quiz questions

#103 
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
Before a review can be done, you will need to fix the formatting issue.
Please run `npm run fix-style`

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
-    Question: "Which of the following is not something BDD (Behavioral Driven Development) focuses on?",
+    Question: "What is the purpose of a burndown chart in Agile development?",
+    Answer: "To track the progress of user stories throughout a sprint.",
+    Distractor1: "To estimate the total cost of a project.",
+    Distractor2: "To assign tasks to team members.",
+    Distractor3: "To prioritize backlog items.",
+    Explanation:
+      "A burndown chart in Agile development is used to track the progress of user stories or tasks throughout a sprint. It visually represents the amount of work remaining over time, helping the team to manage their workload effectively.",
+    Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/"
+  },
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:520]: this is a duplicate.
please remove

File: src/data/agile-quiz.ts [hunk 2]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
+    Question: "What is the purpose of a daily stand-up meeting in Agile?",
+    Answer:
+      "To provide team members with a quick, daily update on progress, identify and address issues, and synchronize their work.",
+    Distractor1: "To discuss long-term project strategy and planning.",
+    Distractor2: "To assign tasks for the entire sprint.",
+    Distractor3: "To review the project's final deliverables.",
+    Explanation:
+      "Daily stand-up meetings in Agile, also known as daily scrums, serve to provide team members with a daily opportunity to share updates on their work, identify any issues, and ensure that the team is aligned and focused on the sprint goal.",
+    Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/"
+  },
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:531]: a similar question already exists in the file.
Please remove

File: src/data/agile-quiz.ts [hunk 3]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
+  {
+    Question: "What is the purpose of a Product Owner in Agile?",
+    Answer:
+      "To represent the customer or stakeholder's interests, prioritize the backlog, and make decisions regarding what features should be developed.",
+    Distractor1: "To manage the technical aspects of the development process.",
+    Distractor2: "To facilitate daily stand-up meetings.",
+    Distractor3: "To write code for the project.",
+    Explanation:
+      "The Product Owner plays a crucial role in Agile by acting as the voice of the customer, ensuring that the team is working on the most valuable features, and making decisions to maximize the project's ROI.",
+    Link: "https://www.freecodecamp.org/news/agile-software-development-handbook/#:~:text=Scrum%20Roles,on%20the%20most%20valuable%20features."
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:541]:
```suggestion
    Link: "https://www.freecodecamp.org/news/agile-software-development-handbook/#scrum-roles"
```

File: src/data/agile-quiz.ts [hunk 4]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
+    Question: "What is the primary goal of a retrospective meeting in Agile?",
+    Answer:
+      "To reflect on the past sprint, identify areas for improvement, and make actionable changes for the next sprint.",
+    Distractor1: "To assign tasks for the upcoming sprint.",
+    Distractor2: "To review and approve project budgets.",
+    Distractor3: "To create the initial project backlog.",
+    Explanation:
+      "Retrospective meetings are essential for continuous improvement in Agile. They provide a structured opportunity for the team to assess their work, discuss what went well and what didn't, and plan for improvements in the next sprint.",
+    Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/"
+  },
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:553]: a similar question already exists in the file.
Please remove

File: src/data/agile-quiz.ts [hunk 5]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
+      "In Agile, which of the following is not part of the standard agenda for Daily Stand-up (DSU)?",
+    Answer: "Deep discussions on architecture or paradigms.",
+    Distractor1: "What you have done since the last DSU.",
+    Distractor2: "What you will be doing after this DSU.",
+    Distractor3:
+      "What are the major obstacles that are stopping your progress, and where do you need help?",
+    Explanation:
+      "The Daily Standup (DSU) or Daily Scrum meeting is a meeting held daily at the same time or in the same location. It's an integral part of Scrum that allows team members to listen to each other's contributions and attempt to identify areas where they can assist each other's progress.",
+    Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/#daily-stand-up-and-daily-scrum"
   },
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:609]: this is a duplicate question
please remove

File: src/data/agile-quiz.ts [hunk 6]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
+    Question:
+      "In Agile, which of the following is not part of the standard agenda for Daily Stand-up (DSU)?",
     Answer: "Deep discussions on architecture or paradigms.",
     Distractor1: "What you have done since the last DSU.",
     Distractor2: "What you will be doing after this DSU.",
-    Distractor3: "What are the major obstacles that are stopping your progress, and where do you need help?",
-    Explanation: "The Daily Standup (DSU) or Daily Scrum meeting is a meeting held daily at the same time or in the same location. It's an integral part of scrum that allows team members to listen to other's contributions and attempt to identify areas where they can assist each other's progress.",
-    Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/#daily-stand-up-and-daily-scrum",
+    Distractor3:
+      "What are the major obstacles that are stopping your progress, and where do you need help?",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:617]:
```suggestion
      "Deep discussions on frameworks and libraries",
```

File: src/data/agile-quiz.ts [hunk 7]
```diff
@@ -490,41 +499,152 @@ const agileQuiz = [
...
+      "A simple, informal description of a software feature from an end user's perspective.",
+    Distractor1:
+      "A comprehensive project plan for the entire development process.",
+    Distractor2: "A technical document used by developers to write code.",
+    Distractor3:
+      "A formal contract between the development team and stakeholders.",
+    Explanation:
+      "User Stories are a common way of expressing software requirements in a way that's easy to understand by both developers and non-technical stakeholders.",
+    Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/"
+  },
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:634]: a similar question already exists in the file.
Please remove

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by SWASTI-SUNDAR** [comment]
i have made all changes as required please review it @jdwilkin4

File: src/data/agile-quiz.ts (hunk 4 above)
Comment (jdwilkin4) [reply src/data/agile-quiz.ts:532]: @SWASTI-SUNDAR 

This duplicate is still here
It needs to be removed

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Pull request merged**